UNL-CSE-AMBASSADORS/VITA
# 118 Production SQL Script

**UPDATE**: I want to discuss with @CollinsSpencer how DST will affect any of these times in the DB. 
**SO DO NOT MERGE**

Finished up the production data SQL script. This contains all the data that we need to manually insert for this year. 
Also changed the permission edit_user_permission to edit_user_permissions

**TO TEST**
Just run the FullDB.sql and then FullDBProdData.sql, and make sure there is no errors.
